src/test/java/it/pagopa/transactions/commands/handlers/v2/TransactionRequestAuthorizationHandlerTest.java

```diff
@@ -404,6 +428,15 @@ void shouldSaveAuthorizationEventNpgCardsRedirectToExternalDomain() {
                 )
         ).thenReturn(Mono.empty());
 
+        Mockito.when(
```

Comment:
add a verify() condition to check that `transactionAuthorizationRequestedQueueAsyncClient.sendMessageWithResponse` is effectively called once with the expected event body, TTL and visibility timeout.
This also applies also to all below junit tests.
Please add also a junit tests with other payment gateway type (such as REDIRECT) and add verifications against the fact that for payment gateways other than NPG this event is not written in the queue

Reply:
Handled in 76aa9d14